### PR TITLE
feat(Hybrid Map): fixed LazyLoading forceCheck and Listing performance

### PR DESCRIPTION
**Pull request by grenti**
affects: @rentpath/react-ui-ag, @rentpath/react-ui-core

[Card](https://rentpath.leankit.com/card/624494614)
- [x] Fixed react-lazyload forcecheck issue
during onSlide
- [x] Added feature to MobileMapListing to only render PhotoCarousel when active
else display background image photo
- [x] Updated MobileMapListing.css for Photo to display
background Photo with appropriate height

### CODE REVIEW

File: packages/react-ui-ag/src/Listings/MobileMapListing.js [hunk 1]
```diff
@@ -99,6 +108,11 @@ export default class MobileMapListing extends Component {
     }
   }
 
+  @autobind
+  handlePhotoCarouselSlide(index) { // eslint-disable-line no-unused-vars
```
Comment (nray621) [inline packages/react-ui-ag/src/Listings/MobileMapListing.js]: I'm guessing there is a reason, but I'm confused why you're giving this function a parameter at all

**Comment by nray621** [comment]
Other than the one comment above, LGTM 👍

**Comment by grenti** [comment]
@nray621 - A grandchild component `Carousel`, is using the handler and it's signature expects an `index` param. Even though we aren't using it here, I want to make it clear that an index is passed into the callback so that future users don't need to dig three levels deep into `<ListComponents.Photos />`, `<PhotoCarousel /> and finally `<Carousel />` to find out what the actual signature of the prop func call should be. 

This is similar to Express.js middleware callback of `(res, req, next) => { }`. Even if you don't use `req` or `res`, you always pass those params since the caller (Express) expects them. Just trying to make clear intent by using some sort of convention since there's a long chain. 

So... what say you mate? 
[] Yay!
[] Nay!!

**Comment by nray621** [comment]
Ahh gotcha. Yeah I figured it was something along those lines. I'm fine either way

**Comment by bethstevic** [comment]
Looks good 👍

**Pull request merged**